packages/jest-runner/src/runTest.ts

```diff
@@ -259,17 +270,19 @@ async function runTestInternal(
   } finally {
     await environment.teardown();
 
+    // @ts-ignore
+    // TODO: clarify this in PR why this is not on the module
     sourcemapSupport.resetRetrieveHandlers();
```

Comment:
This method isn't available on the type.

Reply:
Could you send a PR to definitelytyped adding it?